changelog/@unreleased/pr-600.v2.yml

```
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Make ExternalPublishJarPlugin configuration-cache friendly
```

```suggestion
  description: Make `ExternalPublishJarPlugin` configuration-cache compatible
```

~Is this what it takes for the `classes` task?~

---

Hmm, I think this PR makes `ExternalPublishJarPlugin` compatible with configuration cache in general.

---

changed it to just compatible

---

Updated my suggestion